extensions/amp-analytics/0.1/vendors.js

```diff
@@ -622,6 +622,37 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'segment': {
```

Comment:
We automatically generate a golden file test for this that you will need to update. See https://travis-ci.org/ampproject/amphtml/builds/138287933